src/Services/AbstractQueuedJob.php

```diff
@@ -117,6 +117,14 @@ public function getJobType()
         return QueuedJob::QUEUED;
     }
 
+    /**
+     * @return int|null
+     */
+    public function getRunAsMember()
```

Comment:
I was about to ask for a better docblock until I saw it in the QueuedJob interface. If you omit the docblock entirely here, PHPDoc will inherit it automatically. The way it is written now will override the interface's doc block